### PR TITLE
feat: update C02 project pages and add update project pages script

### PR DESCRIPTION
**Pull request by wgwz**
## Description

Closes: https://github.com/regen-network/regen-registry/issues/1666

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] manually tested (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,15 @@ Running validations:
 $ ./shacl_validate.py
 ```
 
+## Update project pages
+
+```
+$ python3 -m venv venv
+$ . venv/bin/activate
+(venv) $ pip install sqlalchemy psycopg2
```
Comment (blushi) [inline README.md:49]: I have some issues when trying to install psycopg2:
```
ld: library not found for -lssl
      clang: error: linker command failed with exit code 1 (use -v to see invocation)
      error: command '/usr/bin/clang' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> psycopg2
``` 

I didn't dig too much into it, was wondering if you had came across something similar
I'm using python3 installed with homebrew, wondering if that could be related...

Comment (wgwz) [reply README.md:49]: sorry @blushi ! i wrote the package name. i’ll need to update this. it should be installed with:

pip install psycopg2-binary

Comment (blushi) [reply README.md:49]: works now! could you update the README?

Comment (wgwz) [reply README.md]: done!

Comment (wgwz) [reply README.md]: thanks for testing it and catch that

**Review by blushi — APPROVED** [review]

**Pull request merged**